tests/lua-tests/src/controller.lua

```diff
@@ -1,4 +1,7 @@
 
+-- jit off
+local jit = require("jit")
+jit.off()
```

Comment:
Why have to disable jit?

Reply:
Due to security concerns, JIT needs to be turned off；We will consider adapting the third-party library to disable JIT in the future

Reply:
Ok, then it is better to only disable on OHOS and do it in mainmenu.lua.

Reply:
Due to security concerns, JIT needs to be turned off；We will consider adapting the third-party library to disable JIT in the future

Reply:
But current implementation disable it for all platforms.

Reply:
sorry，modified